packages/models/src/queue/messages.ts

```diff
@@ -57,10 +57,25 @@ export const EserviceDto = z.object({
   versionId: z.string().uuid(),
   technology: EserviceTechnology,
   state: EserviceInteropState,
-  basePath: z.array(z.string()).nonempty().max(2048),
+  basePath: z
+    .array(z.preprocess((val) => sanitizeData(val as string), z.string()))
```

Comment:
should this keep the runtime check on string?
```suggestion
    .array(z.coerce.string().transform(sanitizeData))
```